docs/_static/mxnet-theme/navbar.html

```diff
@@ -30,7 +31,7 @@ <h1 id="logo-wrap">
          <span id="dropdown-menu-position-anchor-docs">
           <a href="#" class="main-nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="true">Docs <span class="caret"></span></a>
           <ul id="package-dropdown-menu-docs" class="dropdown-menu navbar-menu">
-            <li><a class="main-nav-link" href="{{url_root}}tutorials/index.html">Tutorials</a></li>
```

Comment:
why remove tutorials?

Reply:
Removing due to redundancy, since "Learn" already takes us to Tutorials.

Reply:
This is the directive from the PM (Seb) as I was working through the SIMs. Also, I see "Learn" nav link takes to Tutorials as well.